Formula/b/bozohttpd.rb

```diff
@@ -0,0 +1,32 @@
+class Bozohttpd < Formula
+  desc "Small and secure http version 1.1 server"
+  homepage "http://www.eterna.com.au/bozohttpd/"
+  url "http://www.eterna.com.au/bozohttpd/bozohttpd-20220517.tar.bz2"
+  sha256 "9bfd0942a0876e5529b0d962ddbcf50473bcf84cf5e4447043e4a0f4ea65597a"
+  license "BSD-2-Clause"
+
+  depends_on "pkg-config" => :build
+  depends_on "lua"
+  depends_on "openssl@3"
+
+  def install
+    lua_libs = system "pkg-config", "--libs", "lua"
+    ENV.append "CFLAGS", lua_libs.to_s
+    lua_cflags = system "pkg-config", "--cflags", "lua"
+    ENV.append "CFLAGS", lua_cflags.to_s
+    libcrypto_libs = system "pkg-config", "--libs", "libcrypto"
+    ENV.append "CFLAGS", libcrypto_libs.to_s
+    libcrypto_cflags = system "pkg-config", "--cflags", "libcrypto"
+    ENV.append "CFLAGS", libcrypto_cflags.to_s
+    ENV.append "CFLAGS", "-O"
+    ENV.append "CFLAGS", "-D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64"
+    ENV.append "CFLAGS", "-D_GNU_SOURCE -D_DEFAULT_SOURCE"
+    ENV.append "CFLAGS", "-DNO_BLOCKLIST_SUPPORT -DDO_HTPASSWD"
+    system "make", "-f", "Makefile.boot", "CC=#{ENV.cc}"
+    bin.install "bozohttpd"
+  end
+
+  test do
+    system "#{bin}/bozohttpd", "-G"
```

Comment:
Can we replicate the test for httpd here?

Reply:
Done